specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -1822,8 +1850,11 @@
           }
         ],
         "responses": {
-          "204": {
-            "description": "Success"
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/WebhookHookInfo"
```

Comment:
Should this be a generic `HookInfo` instead?

Reply:
Yes.

Reply:
Fixed in the latest commit.